.github/workflows/ci.yaml

```diff
@@ -36,6 +36,27 @@ jobs:
       target-branch: ${{ inputs.target-branch }}
     secrets: inherit
 
+  publish-manifest:
+    runs-on: ubuntu-latest
+    needs: [publish-kata-deploy-payload-amd64, publish-kata-deploy-payload-arm64]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Login to Kata Containers quay.io
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push multi-arch manifest
+        run: |
+          docker manifest create ghcr.io/kata-containers/kata-deploy-ci:${{ inputs.tag }} \
+          --amend ghcr.io/kata-containers/kata-deploy-ci:${{ inputs.tag }}-amd64 \
+          --amend ghcr.io/kata-containers/kata-deploy-ci:${{ inputs.tag }}-arm64
+          docker manifest push quay.io/kata-containers/kata-deploy-ci:${{ inputs.tag }}
```

Comment:
Is this a reason that we aren't building teh s390x and ppc64le images into the manifest, like we do in payload-after-push: https://github.com/kata-containers/kata-containers/blob/b056683b7a4e6a57e3526955ff9c7f4cca779450/.github/workflows/payload-after-push.yaml#L104-L111 ?